back-end/services/loansService.js

```diff
@@ -0,0 +1,6 @@
+const { authenticate, checkAccept } = require("./productsService");
```

Comment:
@puteulanus instead of repeating the exports, just rename productsService to authService and use that directly